### PR TITLE
Fix incorrect FreeBSD to Cloudsmith package mapping

### PR DESCRIPTION
**Pull request by Theodus**

### CODE REVIEW

**Comment by ponylang-main** [comment]
Hi @Theodus,

The changelog - fixed label was added to this pull request; all PRs with a changelog label need to have release notes included as part of the PR. If you haven't added release notes already, please do.

Release notes are added by creating a uniquely named file in the `.release-notes` directory. We suggest you call the file `162.md` to match the number of this pull request.

The basic format of the release notes (using markdown) should be:

```
## Title

End user description of changes, why it's important,
problems it solves etc.

If a breaking change, make sure to include 1 or more
examples what code would look like prior to this change
and how to update it to work after this change.
```

Thanks.

File: .release-notes/162.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Add unknown vendor for FreeBSD lookup
```
Comment (SeanTAllen) [inline .release-notes/162.md]:
```suggestion
## Fix incorrect FreeBSD to Cloudsmith package mapping
```

**Comment by SeanTAllen** [comment]
@Theodus I can't commit my own suggestion. Can you take a look and if you agree, accept the update to the release note header?

**Comment by Theodus** [comment]
The header has already been updated

**Pull request merged**